wiki/Tournaments/OMIC/2019_4K/id.md

```diff
@@ -0,0 +1,345 @@
+---
+tags:
+- O!MIC 4K 2019
+- OMIC
+---
+
+# osu!mania 4K Indonesia Cup 2019
+![O!MIC 4K 2019 Banner](img/logo.png)  
+**osu!mania 4K Indonesia Cup 2019** adalah turnamen solo osu!mania 4K khusus wilayah Indonesia dengan memakai sistem pertandingan Royal Rumble pada babak setelah Kualifikasi, lalu satu lawan satu dengan Double-Elimination dimulai dari Penyisihan Grup.
+
+## Jadwal Turnamen
+| Sesi | Tanggal |
+| --: | :-- |
+| Pendaftaran | 26 Mei - 9 Juni 2019 |
+| Screening | 10 - 16 Juni 2019
+| Kualifikasi | 22 - 23 Juni 2019 |
+| Babak R56 | 29 - 30 Juni 2019 |
```

Comment:
```suggestion
| Babak 56 Besar | 29 - 30 Juni 2019 |
```
I'm unsure if you're pointing out Round of 56 (?)
Apply for other formats that use prefix `Round of`